distributed/scheduler.py

```diff
@@ -4995,7 +4995,7 @@ async def remove_worker(
         event_msg["worker"] = address
         self.log_event("all", event_msg)
 
-        logger.info("Remove worker %s", ws)
+        logger.info(f"Remove worker {ws} ({stimulus_id=})")
```

Comment:
nit: This is a micro-optimization, but I'm wondering if adopting https://google.github.io/styleguide/pyguide.html#3101-logging, i.e., avoiding f-strings in logs, as a general best practice would be useful.

Reply:
In general, yes.
For DEBUG level logging that is most frequently disabled, hard yes.
In this specific case, it would be substantially more verbose, so -0.5 from me